bin/local-lib-installer

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 [ -z "$HELPERS_ROOT" ] && export HELPERS_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
 source "$HELPERS_ROOT/lib/debug.sh"
+exec 1>&2 # since local-lib calls this, then evals stdout - any echo to stderr
```

Comment:
Seems like this should be handled in local-lib, and/or selectively for the outputs of this script.

Reply:
Agreed, I've moved it to local-lib and also only added the "lib" there if -installer exits with success.